### PR TITLE
Allow overriding plutus-apps in CI.

### PR DESCRIPTION
**Pull request by shlevy**

### CODE REVIEW

**Comment by shlevy** [comment]
Converted to draft, hydra doesn't give us sha256 so need to tweak this

**Comment by shlevy** [comment]
@silky OK, this is working locally, but it relies on https://github.com/input-output-hk/haskell.nix/pull/1354

**Comment by silky** [comment]
@shlevy ; interesting; this doesn't look too bad, thanks for pulling it together!

Hopefully they merge that :)

File: release.nix [hunk 1]
```diff
@@ -1,9 +1,31 @@
+{ plutus-apps ? null
+}:
+
 # The content of this file was partially copied from the equivalent file in the plutus repository.
 # It is used by IOHK's Hydra for CI (building the project, running the tests, etc.)
 #
 # Therefore, do not worry too much about the structure.
 let
-  packages = import ./.;
+  source-repo-override = if plutus-apps == null then {} else {
```
Comment (silky) [inline release.nix:9]: @shlevy I suppose we could use a comment here explaining what this is doing :) 😅

Comment (shlevy) [reply release.nix:9]: :+1: Will do once the haskell.nix side is covered

**Pull request merged**